subsys/bluetooth/mesh/shell/shell_utils.c

```diff
@@ -48,6 +51,74 @@ bool shell_model_str2bool(const char *str)
 	return str2u8(str);
 }
 
+
+int shell_model_str2sensorval(const char *str, struct sensor_value *out)
```

Comment:
This function looks like `string_conv_str2dbl` from https://github.com/zephyrproject-rtos/zephyr/pull/43289. Once it downstreamed, we should probably use it to convert string to double and then convert double to sensor value like it is done here: https://github.com/nrfconnect/sdk-nrf/blob/ec52e674c4d263e743e1bb2beee3d56eb7ffae8d/tests/subsys/bluetooth/mesh/light_ctrl/src/main.c#L400-L406